election/serializers.py

```diff
@@ -89,9 +89,19 @@ def __init__(self, name, idx, profile, grade):
         self.profile = profile
         self.grade = grade
 
-
 class CandidateSerializer(serializers.Serializer):
     name = serializers.CharField()
     id = serializers.IntegerField(min_value=0)
     profile = serializers.DictField(child=serializers.IntegerField())
     grade = serializers.IntegerField(min_value=0, max_value=settings.MAX_NUM_GRADES)
+    num_votes = serializers.IntegerField(min_value=0)
+
+class LinkSerializer(serializers.Serializer):
+
+    id_election=serializers.CharField()
+    select_language = serializers.CharField(max_length=2,default="en")

```

Comment:
For `select_language`, it's better not having a default value and not a required field. If nothing is provided, it seems coherent to use the select_language from the election instead.